src/node.cc

```diff
@@ -1159,6 +1153,32 @@ Maybe<bool> ProcessEmitDeprecationWarning(Environment* env,
                                    deprecation_code);
 }
 
+static void OnMessage(Local<Message> message, Local<Value> error) {
+  Isolate* isolate = Isolate::GetCurrent();
+  switch (message->ErrorLevel()) {
+    case Isolate::MessageErrorLevel::kMessageWarning: {
+      Environment* env = Environment::GetCurrent(isolate);
+      if (!env) {
+        break;
+      }
+      v8::String::Utf8Value filename(isolate,
+          message->GetScriptOrigin().ResourceName());
+      // (filename):(line) (message)
+      std::stringstream warning;
+      warning << *filename;
+      warning << ":";
+      warning << message->GetLineNumber(env->context()).FromMaybe(-1);
+      warning << " ";
+      v8::String::Utf8Value msg(isolate, message->Get());
+      warning << *msg;
+      ProcessEmitWarningGeneric(env, warning.str().c_str(), "V8");
```

Comment:
We are ignoring the return value of `ProcessEmitWarningGeneric`, is that okay?

Reply:
So probably wrap with `USE()`